iqmotion/iq_devices/iq_module.py

```diff
@@ -20,23 +20,33 @@ class IqModule:
     _DEFAULT_VELOCITY_CLIENT_ENTRY = ""
     _DEFAULT_VOLTS_CLIENT_ENTRY = ""
 
-    def __init__(self, com: Communicator, module_idn=0, clients_path=None):
+    def __init__(self, com: Communicator, 
+                module_idn=0,  
+                clients_path: str=None, 
+                extra_clients: list=None, 
+                module_file_path: str=None):
         
         self._client_dict = {}
         self._com = com
         self._module_idn = module_idn
 
         # Load the Module JSON (Base, Vertiq, Fortiq, etc...)
-        self._module_file_dict = IqModuleJsonParser.parse_default_modules(
-            self._MODULE_FILE_NAME
-        )
-        
-        # Deprecated: Add a custom module 
```

Comment:
Were custom modules supposed to be deprecated? Why are they making a comeback?

Reply:
I was unaware that it was deprecated. The Pulsing Module was created by Luca and it inherits from CustomIqModule

Reply:
Is it necessary for the Pulsing Module to inherit from CustomIqModule? Why does it need it? I know Luca set it up originally, but if what he did is not quite right then maybe we should change up how it is?

Reply:
I removed the custom_iq_module and pulsing_module and associated files/tests for now. I will discuss with Luca about refactoring PulsingModule and make a new PR for it,